core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.ts

```diff
@@ -41,20 +42,22 @@ export class EmaContentletToolsComponent implements OnChanges {
     @ViewChild('menu') menu: Menu;
     @ViewChild('menuVTL') menuVTL: Menu;
     @ViewChild('dragImage') dragImage: ElementRef;
-    private dotMessageService = inject(DotMessageService);
 
-    private buttonPosition: 'after' | 'before' = 'after';
+    @HostBinding('class.hide') @Input() hide = false;
 
     @Input() contentletArea: ContentletArea;
-    @HostBinding('class.hide') @Input() hide = false;
+    @Input() isEnterprise: boolean;
+
     @Output() addContent = new EventEmitter<ActionPayload>();
     @Output() addForm = new EventEmitter<ActionPayload>();
     @Output() addWidget = new EventEmitter<ActionPayload>();
     @Output() edit = new EventEmitter<ActionPayload>();
     @Output() editVTL = new EventEmitter<VTLFile>();
     @Output() delete = new EventEmitter<ActionPayload>();
 
-    items: MenuItem[] = [
+    private dotMessageService = inject(DotMessageService);
+    private buttonPosition: 'after' | 'before' = 'after';
```

Comment:
@rjvelazco According to recent discussions, we should use native private attributes by ES6 with `#`.

Reply:
Right!